buildozer/default.spec

```diff
@@ -146,6 +146,9 @@ fullscreen = 0
 # bootstrap)
 #android.gradle_dependencies =
 
+# (str) build type ('auto' - default/auto detect, 'ant' - use ant, 'gradle' - use gradle)
+#android.java_build_tool = 'auto'
```

Comment:
Am I right in thinking that `auto` should not actually have the `'` around it here?